data/amanpreet-dev.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Aman Preet Singh",
+  "type": "personal",
+  "bio": "**Passionate developer** and **software crafter** with a passion for **problem-solving**. Always learning and growing in the field.",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AmanpreetDev",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/amanpreet-dev/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/amanpreet.dev/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name" : "Collborate with me on Github",
```

Comment:
You may want to correct the minor typo in the spelling of **' Collaborate '** :+1: 
Rest everything looks good, thank you :tada: 
```Suggestion
      "name" : "Collaborate with me on Github",
```